src/Common/tests/XunitTraitsDiscoverers/Discoverers/ActiveIssueDiscoverer.cs

```
@@ -23,7 +23,7 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
         public IEnumerable<KeyValuePair<string, string>> GetTraits(IAttributeInfo traitAttribute)
         {
             IEnumerable<object> ctorArgs = traitAttribute.GetConstructorArguments();
-            Contract.Assert(ctorArgs.Count() >= 2);
+            Debug.Assert(ctorArgs.Count() >= 2);
```

This is gonna be nightmare to debug if it fails fast